utilities/test_suite/HIP/runTests.py

```diff
@@ -34,6 +34,10 @@
 inFilePath1 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
 inFilePath2 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src2')
 qaInputFile = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
+# Use input images of same dimensions for RICAP Testcase 82
```

Comment:
Was this set as a requirement from the ricap paper?

Reply:
Yeah from the documentation Swetha had shared and the repo link this was what I understood and confirmed the same with her too.

Reply:
As per paper they had resized all images to one common size 416x416 and experimented with them.